layers/+tools/meson/packages.el

```diff
@@ -0,0 +1,24 @@
+;;; packages.el --- meson layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Matthew Leach <dev@mattleach.net>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary: A simple layer that wraps meson-mode
```

Comment:
Please remove the commentary header, this should go into the README.org if it is not already there.